README.rst

```diff
@@ -34,6 +34,8 @@ Market-oriented information on SDC from Dräger can be found `here <https://www.
 
 SDC is supported by the interoperability association `OR.Net <https://ornet.org/en>`__.
 
+For this open source project the `Contributor Licence Agreement <https://github.com/Draegerwerk/sdc11073/Contributor_License_Agreement.md>`__ governs all relevant activities and your contributions. By contributing to the project you agree to be bound by this Agreement and to licence your work accordingly.
```

Comment:
It appears this link would not work. I tried to use the same kind of link with another file name of a file that already exists on the master branch and it did not work.

Perhaps it should be changed according to the following suggestion?

```suggestion
For this open source project the `Contributor Licence Agreement <https://github.com/Draegerwerk/sdc11073/blob/master/Contributor_License_Agreement.md>`__ governs all relevant activities and your contributions. By contributing to the project you agree to be bound by this Agreement and to licence your work accordingly.
```

Reply:
done